src/index/IndexBuilderMain.cpp

```diff
@@ -266,7 +267,12 @@ int main(int argc, char** argv) {
       // vocabulary for  text  index  creation)
 
       if (ntFile.size() > 0) {
-        index.createFromFile<NTriplesParser>(ntFile, allPermutations);
+        if (ad_utility::endsWith(ntFile, ".ttl") ||
```

Comment:
I'm not sure we want to do this based on the file ending. I'm pretty sure I've seen NTriple files that mistakenly had the `.ttl` ending somewhere on our filesystem. Also it's a break from using `-n` for NTriple and `-t` for TSV